engine-rest/engine-rest/src/test/java/org/camunda/bpm/engine/rest/ExternalTaskRestServiceInteractionTest.java

```diff
@@ -399,6 +401,54 @@ public void testFetchAndLockWithCreateTimeDesc() {
     verifyNoMoreInteractions(fetchAndLockBuilder, fetchTopicBuilder, externalTaskService);
   }
 
+  @Test
+  public void testFetchAndLockWithCreateTimeDescCaseInsensitive() {
```

Comment:
If the [point](https://github.com/camunda/camunda-bpm-platform/issues/4051#:~:text=The%20standard%20behaviour,%2C%20sortOrder%20parameters.) for making the sort by createTime API _similar_ to other APIs is true, this method should be adjusted to expected a **bad request**